gnat2goto/driver/gnat2goto_itypes.adb

```diff
@@ -96,13 +98,36 @@ package body Gnat2goto_Itypes is
                                                     "Unknown Ekind"));
    end Do_Itype_Definition;
 
+   ----------------------------
+   -- Do_Itype_Array_Type --
+   ----------------------------
+
+   function Do_Itype_Array_Type (E : Entity_Id) return Irep is
+      Var : constant Node_Id :=
```

Comment:
This should get a better name than `Var`. I don’t know, something like `Type_Of_Underlyign_Array` or something of the sort?